server/models/signers_stats.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import (
+	"github.com/gochain-io/gochain/v3/common"
+)
+
+type SignerStats struct {
+	Name        string
+	URL         string
+	Region      string
+	Signer      common.Address `json:"signer"`
+	BlocksCount int            `json:"blocks_count"`
```

Comment:
I was thinking that these Node fields could be separate so that they aren't duplicated here, and because we'll want them to be available system wide to use for other pages. Since it's static, it could be loaded once and kept on hand. Then this struct can be dropped, and just be a `map[common.Address]uint64` instead.

Reply:
The _code_ is a little less duplicated this way, but I was talking about _data_ duplication - each set of stats contains all of the signers' `Node`s, and this information is sent over the wire every time, even though it is completely static. Additionally, other pages in the system will need this same data, so it would be much more efficient to send it once and save it in the front end to have on hand for any page that might contain an address.

Reply:
Caching on frontend makes sense, I'll discuss with @bergauz